lib/config/validation.spec.ts

```diff
@@ -521,27 +521,6 @@ describe('config/validation', () => {
       expect(errors).toHaveLength(0);
     });
 
-    it('errors if aliases depth is more than 1', async () => {
```

Comment:
why?

Reply:
Before there wasn't any type definition for aliases in `RenovateConfig` so compiler would allow this test but now it throws Type Error. 
Maybe we can update log messages which are returned for invalid config. Currently we handle only invalid uri. We can extend it to handle the extra depth too.

Reply:
I couldn't think of a workaround to keep the test case without failing linting.

Reply:
We should keep it and adjust types if necessary